<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="rapp_core_agent"/>

  <arg name="audio_play_server" default="true"/>
  <arg name="audio_record_server" default="true"/>

  <include if="$(arg audio_play_server)"
    file="$(find rapp_core_agent)/launch/audio_play_server.launch">
    <arg name="namespace" value="$(arg namespace)"/>
  </include>

  <include if="$(arg audio_record_server)"
    file="$(find rapp_core_agent)/launch/audio_record_server.launch">
    <arg name="namespace" value="$(arg namespace)"/>
  </include>

  <node
    type="core_agent"
    name="rapp_core_agent"
    pkg="rapp_core_agent"
    respawn="true"
    respawn_delay="10"
    output="screen">

  </node>


</launch>
